.travis.yml

```diff
@@ -43,15 +43,20 @@ install:
 env:
   - PART="theano/tests/test_flake8.py" DOC=1
   - PART="-e test_flake8.py theano/compat theano/compile theano/d3viz theano/gof theano/misc theano/sandbox theano/scalar theano/scan_module theano/sparse theano/tests theano/typed_list"
-  - PART="theano/tensor"
+  - PART="theano/tensor -e theano/tensor/tests/test_basic.py"
+  - PART="theano/tensor/tests/test_basic.py"
 
 matrix:
   fast_finish: true
   include:
     - python: "3.3"
-      env: PART="-e test_flake8.py ." THEANO_FLAGS="mode=FAST_COMPILE"
+      env:
+        - PART="-e theano/tensor test_flake8.py ." THEANO_FLAGS="mode=FAST_COMPILE"
+        - PART="theano/tensor" THEANO_FLAGS="mode=FAST_COMPILE"
```

Comment:
This won't do what we want I think. I think it should be:

        - PART="-e theano/tensor -e test_flake8.py ." THEANO_FLAGS="mode=FAST_COMPILE"
        - PART="theano/tensor -e test_flake8.py" THEANO_FLAGS="mode=FAST_COMPILE"

I'm not sure we can pass many -e parameter to nosetests, double check that.

Also, line 46 give the full path to test_basic.py, while here we pass only the file name. Make sure that passing the full path is valid. You can check in git log of this file, we where doing that in the path.